### ISSUE TITLE
Applying a Style expands modules

### ISSUE THREAD

**Issue by StudioPetrikas** [aurelienpierreeng/ansel#13]
Applying an image style to an original photo applies the style and expands seemingly random modules. (Most often, "color balance rgb")

This issue is unique to R&Darktable, since in Darktable only one module can be expanded at a time. 

While this isn't a severe issue, but if a style contains a stack of color balance rgb modules, with masks, the module list becomes and endless stream of expanded color balance rgb modules. 

The workflow becomes:
1. Click on an original image
2. Apply a style
3. Scroll/Aim for a module name (any) and Shift+click to collapse all, except for the selected module. 
4. Hope that the focused module isn't "color balance rgb" since it take the whole height of the screen anyway
5. Shift+Click on the module you want to work on. 

Whereas the workflow should be:
1. Click on an original image
2. Apply a style
3. Shif+Click on a collapsed module in the stack to modify the values

**To Reproduce**
1. Create a development with 5-6 "color balance rgb" modules
2. Create a style and include all the "color balance rgb" modules (collapse them all, or don't, doesn't matter)
3. Select any original photo
4. Apply the style from the Darkroom view (bottom left corner)
5. Should result in too many modules expanded.

**Expected behavior**
...
...
4. Apply the style from the Darkroom view (bottom left corner)
5. Style applied with all modules collapsed (ideally)

**Screencast**
https://www.youtube.com/watch?v=cFaKLzjJLaw
(don't know what happened to the mouse pointer)

**Which commit introduced the error**
R&Darktable 3.9.0+1641~g63deb9862

**Platform**
_Please fill as much information as possible in the list given below. Please state "unknown" where you do not know the answer and remove any sections that are not applicable _

* darktable version : 3.9.0+1641~g63deb9862
* OS                : Win11 Build 22000

**Comment by ptilopteri** [comment]
as you found with scrolling, this is also configurable

**Comment by StudioPetrikas** [comment]
> as you found with scrolling, this is also configurable

Please show me where so I can close/remove this. 
I've went through every settings line multiple times and were unable to find such setting.

**Comment by ptilopteri** [comment]
try:  config -> lighttable -> expand a single utility monule at a time

**Comment by StudioPetrikas** [comment]
> try: config -> lighttable -> expand a single utility monule at a time

Yes, I've tried this; it only works in lighttable, and not darktable where the issue appears.

**Comment by RawConvert** [comment]
I get similar behaviour, but I'm mainly using mainstream dt at present, it's happening there.  For me it tends to be the Watermark and Diffuse & Sharpen modules that are expanded.  I just had a look at my Style and interestingly, where modules are listed at the start, there are multiple entries for Watermark and "Diffuse".  Attaching my exported Style, see the first 7 lines
[Style48-dtstyle-renamed-to-dot-TXT.txt](https://github.com/aurelienpierre/R-Darktable/files/9079758/Style48-dtstyle-renamed-to-dot-TXT.txt)
.

**Comment by aurelienpierre** [comment]
Fun and weird.

**Comment by pedrorrodriguez** [comment]
Status?

**Comment by aurelienpierre** [comment]
Fixed
